ocfweb/docs/templates/docs/partials/account-policies.html

```diff
@@ -0,0 +1,31 @@
+        <li>Failing to respond to an account compromise</li>
+        <li>Leaving the account vulnerable to security threats</li>
+        <li>Causing excessive disruption to OCF services, whether intentional or not</li>
+        <li>Neglecting to use the account for an extended period of time</li>
+    </ol>
+</p>
+<p>Misuse of lab, computer, networking, or information resources may result in the loss of computing or lab privileges. Users may be held accountable for their conduct under any applicable University or campus policies, procedures, or collective bargaining agreements. Illegal reproduction of software protected by U.S. Copyright Law is subject to civil damages and criminal penalties including fines and imprisonment.</p>
+<p><strong>Examples of misuse</strong> include, but are not limited to:
+    <ol>
+        <li>Using or allowing someone else to use a computer account that you are not authorized to use, such as by obtaining a password for a computer account without the consent of the account owner, or by sharing your account password.</li>
+        <li>Using the campus network to gain unauthorized access to any computer systems.</li>
+        <li>Knowingly performing an act which will interfere with the normal operation of computers, terminals, peripherals, or networks.</li>
+        <li>Knowingly running or installing on any computer system or network, or giving to another user, a program intended to damage or to place excessive load on a computer system or network. This includes but is not limited to programs known as computer viruses, Trojan horses, and worms.</li>
+        <li>Attempting to circumvent data protection schemes or uncovering security loopholes.</li>
+        <li>Violating terms of applicable software licensing agreements or copyright laws.</li>
+        <li>Using electronic mail or communication to harass other users or send bulk unsolicited communications.</li>
+        <li>Posting materials on electronic bulletin boards that violate existing laws or the University's codes of conduct.</li>
+        <li>Attempting to monitor or tamper with another user's electronic communications, or reading, copying, changing, or deleting another user's files or software without the explicit agreement of the owner.</li>
+        <li>Using OCF or University equipment or resources for commercial purposes unless specifically authorized.</li>
+        <li>Failing to follow common rules of etiquette while in the lab or while using OCF services.</li>
+    </ol>
+</p>
+<p>Violations of this policy or any other OCF policy may result in account suspension. Additionally, misuse may result in suspension or termination of computing privileges. Furthermore, OCF staff may disable an account and/or ban a person from the OCF if there exist exigent circumstances which require such actions.</p>
+<p>The banning of any person from the OCF will be subject to the <a href="/docs/membership/banning/">User Banning Policy</a>.</p>
```

Comment:
I think we should if the lines are this long and probably should be doing it for other files too. It just makes editing and reading a lot easier, and if it doesn't affect the page's layout (and it shouldn't with HTML), then I think it's a good idea.